README.md

```diff
@@ -63,6 +63,8 @@ pod install
 
 ### Handle the redirect
 
+**Important**: This is needed if you want to support iOS 10 and older. Starting from iOS 11 Okta uses modern API which does not implies redirects to/from browser.
```

Comment:
> does not implies redirect to/from browser

Can you elaborate on this? I'm having trouble understanding what this means, as we're still utilizing redirects for iOS 11+

Reply:
Rewritten.